SINGLE_CONNECTION.md

```diff
@@ -0,0 +1,195 @@
+idle-timeout=number | The maximum time (in seconds) which indicates how long idle connections are left open. Although the IDLE_CONNECTION_TIMEOUT system property is a global setting for timing out idle connections, you can use this connection property to override the system property on a per connection basis. By default, idle-timeout is set to -1, meaning that the global setting is used. If you set a value greater than 0, this property overrides any value set for IDLE_CONNECTION_TIMEOUT. You may also set a value of 0 to specifically disable this property for the connection. Any time a client connection is terminated for being idle for too long, a message will be logged under the net category.
+isolation=level | The default transaction isolation level for the connection to use. Isolation levels supported are consistent_read and read_committed. Note that these values are case sensitive.
+keepAlive=[ true \| false ] | Allows enabling of TCP KeepAlive on the connection from client to Transaction Engine. If not specified, keepAlive defaults to false. When keepAlive=true, the driver enables TCP KeepAlive on the socket that it opens to the Transaction Engine. 
+LBPolicy=valueA | From the client application, this property allows selective load balancing across Transaction Engines. For information on using LBPolicy, see Load Balancer Policies. 
+LBQuery=\<selector>(\<filter>) | From the client application, this property allows selective load balancing across Transaction Engines. For information on using LBQuery, see Load Balancer Policies.
+lock-wait-timeout=sec | Configures lock wait time (in seconds) when establishing a new connection.
+memory-limit-bytes=size | The memory limit (per connection) in bytes for all blocking SQL engine operations: Hash-grouping, sorting, sorting via priority queue for limit, distinct (via hash-grouping), union, listagg, table functions, and stored procedures returning result set(s) that accumulate data - in main memory. To run statements with a memory limit different from the system-wide limit, start a new connection with this connection property set to the per-connection limit. If you do not define a value for memory-limit-bytes, the value set for the DEFAULT_CONNECTION_MEMORY_LIMIT system property is the default setting used.
+password=pwd | The password for the user account used to connect to the database. 
+PreferInternalAddress=[ true \| false ] | Use internal address and port for TE, as specified by the external-address and external-port process labels. Default is false (disabled). For a detailed explanation, see Controlling Connectivity, a special case of using Load Balancer Policies with labels. 
```

Comment:
"For a detailed explanation, see Controlling Connectivity, a special case of using Load Balancer Policies with labels. "
can be removed cause we don't have a link

Reply:
Added link